src/main/webapp/JS/regValidation.js

```diff
@@ -0,0 +1,164 @@
+
+var miniUserNameLength = 5;
+var minPassLength = 8;
+
+//password strength validator
+password.addEventListener('input', function()
+{
+
+      var password = document.getElementById('password');
+      var meter = document.getElementById('password-strength-meter');
+      var text = document.getElementById('password-strength-text');
+      var passwordVal = password.value;
+      var result = 0;
+      var strength = {
+        0: "Worst",
+        1: "Bad",
+        2: "Weak",
+        3: "Good",
+        4: "Strong",
+        5: "Very Strong"
+      }
+
+      //if cintains lowercase
+      if(/[a-z]/.test(passwordVal)){
+        result+=1;
+      }
+      //if contains uppercase
+      if(/[A-Z]/.test(passwordVal)){
+        result+=1;
+      }
+      //if conteins special characters
+
+      if(/[.!@#$%^&*()_+-]/.test(passwordVal)){
+        result+=1;
+      }
+      //if contains number
+      if(/[\d]/.test(passwordVal)){
+        result+=1;
+      }
+      //if is longer than 8 characters
+      if(passwordVal.length >=8){
+        result+=1;
+      }
+
+      if(result>5){
+      result=5;
+      }
+
+      // Update the password strength meter
+      meter.value = result;
+
+      // Update the text indicator
+      if (passwordVal !== "") {
+        text.innerHTML = "Password Strength: " + strength[result];
+      } else {
+        text.innerHTML = "";
+      }
+    });
+//password match validator
+function ValidatePassMatch(field_id,password,password2)
+{
+
+var passVal = document.getElementById(password).value.trim();
+var pass2Val = document.getElementById(password2).value.trim();
+var field_div = document.getElementById(field_id).parentNode.id;
+
+if(passVal !== pass2Val)
+{
+removeWarning(field_id,field_div);
```

Comment:
Tu mi brakuje wcięć! Bardzo poważna rzecz.
Tym bardziej się to rzuca w oczy, że w funkcji wyżej te wcięcia są.